src/lando/main/sentry.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+import sentry_sdk
+from django.conf import settings
+from sentry_sdk.integrations.django import DjangoIntegration
+
+from lando.version import version
+
+logger = logging.getLogger(__name__)
+
+
+def init_sentry():
+    """Initialize Sentry integration for remote environments."""
+    if not settings.ENVIRONMENT.is_remote:
+        raise ValueError(
+            "Attempting to initialize Sentry in non-remote environment "
+            f"({settings.ENVIRONMENT})"
+        )
```

Comment:
Nit: we also check this in the caller, it can be removed.

Reply:
Changed up the implementation slightly to simplify this.